contracts/facets/ThirdPartyFacet.sol

```diff
@@ -28,185 +28,174 @@ contract ThirdPartyFacet {
     /// @notice Close a sarcophagus that has not been unwrapped within its grace period
     /// @param sarcoId The identifier of the sarcophagus to clean
     /// @param paymentAddress The address to which rewards will be sent
-    function clean(bytes32 sarcoId, address paymentAddress) external {
-        LibUtils.revertIfNotExistOrInactive(sarcoId);
-
-        LibTypes.Sarcophagus storage sarco = s.sarcophagi[sarcoId];
-
-        // Make sure the sarco is cleanable
-        if (block.timestamp < s.gracePeriod + sarco.resurrectionTime) {
-            revert LibErrors.SarcophagusNotCleanable();
-        }
-
-        // Figure out which archaeoligists did not fulfil their duties;
-        // accumulate their digging fees
-        address[] memory archAddresses = sarco.archaeologists;
-
-        uint256 totalDiggingFee;
-
-        for (uint256 i = 0; i < archAddresses.length; i++) {
-            bool didNotUnwrap = s.archaeologistSarcoSuccesses[archAddresses[i]][
-                sarcoId
-            ] == false;
-
-            if (didNotUnwrap) {
-                LibTypes.ArchaeologistStorage memory defaulter = s
-                    .sarcophagusArchaeologists[sarcoId][archAddresses[i]];
-
-                totalDiggingFee += defaulter.diggingFee;
-
-                // decrease the defaulter's cursed bond
-                LibBonds.decreaseCursedBond(archAddresses[i], defaulter.diggingFee);
-
-                // Save the failure to unwrap against the archaeologist
-                s.archaeologistCleanups[archAddresses[i]].push(sarcoId);
-            }
-        }
-
-        (
-            uint256 cleanerBondReward,
-            uint256 embalmerBondReward
-        ) = _distributeLoot(
-                paymentAddress,
-                sarco,
-                totalDiggingFee
-            );
-
-        sarco.state = LibTypes.SarcophagusState.Cleaned;
-
-        emit CleanUpSarcophagus(
-            sarcoId,
-            msg.sender,
-            cleanerBondReward,
-            embalmerBondReward
-        );
-    }
+//    function clean(bytes32 sarcoId, address paymentAddress) external {
```

Comment:
Commented out the entire function instead of updating it to work with the new data structures because it's getting rethought